Flow.Launcher/ViewModel/PluginViewModel.cs

```diff
@@ -30,9 +30,20 @@ public bool PluginState
             get => !PluginPair.Metadata.Disabled;
             set => PluginPair.Metadata.Disabled = !value;
         }
+        public bool IsExpanded
+        {
+            get => _isExpanded;
+            set
+            {
+                _isExpanded = value;
+                OnPropertyChanged();
+                OnPropertyChanged(nameof(SettingControl));
+            }
+        }
 
         private Control _settingControl;
-        public Control SettingControl => _settingControl ??= PluginPair.Plugin is not ISettingProvider settingProvider ? new Control() : settingProvider.CreateSettingPanel();
+        private bool _isExpanded;
+        public Control SettingControl => IsExpanded ? _settingControl ??= PluginPair.Plugin is not ISettingProvider settingProvider ? new Control() : settingProvider.CreateSettingPanel() : null;
```

Comment:
@taooceros this statement is doing my head in, what's it for?

Reply:
Unless the panel is expanded, let's not load the usercontrol.